query-engine/connectors/sql-query-connector/src/query_ext.rs

```diff
@@ -48,14 +48,16 @@ pub trait QueryExt: Queryable + Send + Sync {
         let params: Vec<_> = params.into_iter().map(convert_lossy).collect();
         let result_set = AssertUnwindSafe(self.query_raw(&q, &params)).catch_unwind().await??;
 
+        // Somehow in "certain" cases, query_raw does not return the column names in the result
```

Comment:
If you can't specify more what the problem likely is in the comment, keep it simple.
`// `query_raw` does not always return the column names in the result, so we need to work around it by ...`

Reply:
Cool, I will be more explicit